Pod/Classes/UITextField+Extensions.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  UITextField+Extensions.swift
+//  Pods
+//
+//  Created by Christopher Jones on 3/24/16.
+//
+//
+
+import UIKit
+
+internal extension UITextField {
+
+    internal static var emptyTextFieldCharacter: String {
+        return "\u{202F}"
```

Comment:
Could we put that entire explanation into a comment above Daniel?

On Thu, Mar 24, 2016 at 2:30 PM, Michael Campbell <notifications@github.com>
wrote:

> In Pod/Classes/UITextField+Extensions.swift
> <https://github.com/prolificinteractive/Caishen/pull/10#discussion_r57391096>
> :
>
> > @@ -0,0 +1,18 @@
> > +//
> > +//  UITextField+Extensions.swift
> > +//  Pods
> > +//
> > +//  Created by Christopher Jones on 3/24/16.
> > +//
> > +//
> > +
> > +import UIKit
> > +
> > +internal extension UITextField {
> > +
> > +    internal static var emptyTextFieldCharacter: String {
> > +        return "\u{202F}"
>
> wow, that's interesting
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/prolificinteractive/Caishen/pull/10/files/ea668a9b8b4b5035bfd4e1f0fe623d4e71911fd9#r57391096>
>



-- 

Photis Patriotis

VP of Engineering

Prolific Interactive

BK // 45 Main Street, Suite 1006 | Brooklyn, NY 11201

SF // 995 Market Street, Suite 1401 | San Francisco, CA 94103

m: 215.480.9759 | prolificinteractive.com | @weareprolific
<http://www.twitter.com/weareprolific>